lib/appsignal/utils.rb

```diff
@@ -14,12 +14,81 @@ def sanitize(params, only_top_level = false, key_sanitizer = nil)
     end
     extend ClassMethods
 
+    def self.data_generate(body)
+      Data.generate(body)
+    end
+
+    class Data
+      class << self
```

Comment:
I think I tried a module but it didn't work in the same way. Not sure. We can refactor this when we move the utils to their own file.